src/game/default/g_cmd.c

```diff
@@ -728,6 +722,7 @@ static const char *vote_cmds[] = {
 	"g_rounds",
 	"g_spawn_farthest",
 	"g_teams",
+	"g_num_teams",
```

Comment:
I see no reason to not keep `g_teams` but let folks set it to `1` or `2` for 2 teams, and `3` or `4` for 3 and 4 respectively. `g_num_teams 2` is just less obvious than `g_teams 1` for the 99% use case.